src/components/SectionNotification/SectionNotification.driver.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import { SECTION_NOTIFICATION_DATA_HOOKS } from './dataHooks';
+import style from './SectionNotification.st.css';
+
+export interface SectionNotificationDriver extends BaseUniDriver {
+  isError(): Promise<boolean>;
+  isMobile(): Promise<boolean>;
+  isRTL(): Promise<boolean>;
+  hasIcon(): Promise<boolean>;
+  hasControls(): Promise<boolean>;
+  getText(): Promise<string>;
+  getIconContainer(): UniDriver;
+  getControlsContainer(): UniDriver;
+}
+
+export const sectionNotificationDriverFactory = (
+  base: UniDriver,
+): SectionNotificationDriver => {
+  const stylableUtil = new StylableUnidriverUtil(style);
```

Comment:
Very nice driver
However, we prefer to to not use `StylableUnidriverUtil` since it may cause problems with certain tests.
I believe we removed this from our generator, did you use `npm run generate` to create this new component?

In any case, instead of using `StylableUnidriverUtil`, you can use `data-` attributes in order to test the styling state of the component.
For instance you can see what we have in https://github.com/wix/wix-ui-tpa/blob/master/src/components/Tabs/Tabs.driver.ts